agora/README.md

```diff
@@ -1,3 +1,15 @@
-## Contoso Hypermarket
+# Contoso Hypermarket
```

Comment:
Maybe we could have a folder named contoso-hypermarket that we can have the instrucitons and docker compose?

Reply:
I will move them.